src/SumoCoders/FrameworkCoreBundle/Resources/assets/js/Framework/Form.js

```diff
@@ -1,72 +1,76 @@
 import {Locale} from 'Framework/Locale'
 
 const locale = new Locale()
-const dateFieldOptions = {
-  currentText: locale.get('datepicker.buttons.today'),
-  closeText: locale.get('datepicker.buttons.close'),
-  nextText: locale.get('datepicker.buttons.next'),
-  prevText: locale.get('datepicker.buttons.previous'),
-  firstDay: 1,
-  hideIfNoPrevNext: true,
-  showAnim: 'slideDown',
-  zIndex: 9999,
-  dateFormat: 'dd/mm/yy',
-  dayNames: [
-    locale.get('datepicker.full.days.sunday'),
-    locale.get('datepicker.full.days.monday'),
-    locale.get('datepicker.full.days.tuesday'),
-    locale.get('datepicker.full.days.Wednesday'),
-    locale.get('datepicker.full.days.thursday'),
-    locale.get('datepicker.full.days.friday'),
-    locale.get('datepicker.full.days.saturday')
+
+let capitalizeWord = function (str) {
```

Comment:
does this js get compiled? cause if it isn't the let will break things in some browsers

Reply:
also, are you sure there isn't already a function for this somewhere in the js?